scripts/security/certs-create-per-user.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+i=$1
+
+echo "------------------------------- $i -------------------------------"
```

Comment:
Very Nit: but as a standalone script this line isn't very helpful.  I would probably leave it to the caller to create a printed visual separator in it's loop.  But I wouldn't hold up the PR over it and I will approve.

Reply:
@rspurgeon https://github.com/confluentinc/cp-demo/pull/192/commits/60705971432d267cc2414a64bc18bc4f2693c9a8 addresses this.